app.py

```diff
@@ -231,6 +231,12 @@ def search():
 @app.route("/createEvent.html", methods=['POST','GET'])
 @is_logged_in
 def createEvent():
+    with engine.begin() as conn:
+        queryGetMyEvents = sqlalchemy.text(f'SELECT COUNT (*) FROM events, created_events WHERE created_events.user_id = {session["userid"]} AND events.id = created_events.event_id AND CURRENT_TIMESTAMP < events.date')
+        result = conn.execute(queryGetMyEvents).first()
```

Comment:
Rather than creating multiple variables, you could probably get the data directly into result by doing
'result = conn.execute(queryGetMyEvents).first[0]' all in one line.